### ISSUE TITLE
peptide deleted while running on PPI complex

### ISSUE THREAD

**Issue by kenndshi** [ohuelab/Solubility_AfDesign#4]
I have installed Solubility_AfDesign locally and want to generate a peptide for my protein-protein complex which consists of A chain with 120 amino acids and my second peptide with 14 peptides (B) long. 
When I set the receptor chain as A chain and mention the hotspots considering that the generated peptide will be inside my PPI complex, after running the script it returns a complex with deleted B peptide and generates a new peptide instead of it.
Please help me in solving this error

**Comment by blacktanktop** [comment]
This tool designs a solubility-aware binder for a target protein.
Therefore, if you specify a target protein A and a binder length 14, a new binder of length 14aa is hallucinated with the target protein, and a PDB file of the complex is generated. hotspot may be able to consider the position of hallucination to some extent. The hotspot may be able to take into account the position of the hallucination to some extent.

Perhaps you are expecting another peptide to be inserted at the interface of the target protein A peptide B complex, but that is not the case.

So, it is better to show computationally or confirm experimentally that the peptide outputted this time has higher binding affinity, dG, etc. than the B peptide.
(We believe that it is not possible to hallucinate a good peptide in a single try, and that it is important to do this multiple times to find the best sample).

**Comment by kenndshi** [comment]
Yes, the thing I wanted was to generate a PPI peptide modulator and thought that it could be done with your code.
It seems like it isn't the case and the code wasn't designed for, nevertheless I wonder if I could select 2 chains as a receptor and try that way..
